2018/FOSDEM.yml

```diff
@@ -11,3 +7,1137 @@ description: |
 
   Additional details about the conference are available at
   <https://fosdem.org/2018/>.
+location: Brussels, Belgium
+tags: oVirt fedora rdo
+start: 2018-02-03
+end: 2018-02-04
+talks:
+
+- title: Optimizing software defined storage for the age of flash
+  speaker: Krutika Dhananjay, Manoj Pillai, Raghavendra Gowdappa
+  start: 2018-02-03 18:00 CET
+  end: 2018-02-03 18:50 CET
+  room: Janson
+  track: Performance
+  description: |
+    High-performance flash storage like NVMe SSDs are becoming commonplace
+    in production environments. In this age of fast storage, inefficiencies
+    in storage software stacks that were previously acceptable with slow
+    spinning devices, are no longer tolerable. There are even faster
+    technologies, like NVDIMMs, on the horizon. These developments are
+    particularly challenging for distributed storage solutions like Gluster
+    and Ceph, where network latencies are a fact of life. This talk is on
+    performance analysis and troubleshooting tools and techniques for this
+    age.
+
+- title: 'Python 3: 10 years later (looking back at Python evolutions of the last
+    10 years)'
+  speaker: Victor Stinner
+  start: 2018-02-03 11:00 CET
+  end: 2018-02-03 11:50 CET
+  room: K.1.105 / La Fontaine
+  track: Python
+  description: |
+    Python 3.0 was released 10 years ago. It's time to look back: analyze
+    the migration from Python 2 to Python 3, see the progress we made on
+    the language, list bugs by cannot be fixed in Python 2 because of the
+    backward compatibility, and discuss if it's time or not to bury Python
+    2.
+
+    Python became the defacto language in the scientific world and the
+    favorite programming language as the first language to learn
+    programming.
+
+- title: 'Deploying OpenStack in the opnv community: cross community ci'
```

Comment:
This should be OPNFV (upper case) - I corrected the typo in the talk title in Penta, so this may update automatically. Also, CI should be caps.

Reply:
This data comes directly from FOSDEM… it should be fixed there.

That said, I can add it to the rewrite list and send a new PR with our local rewrite fix.

Reply:
Thanks - I fixed it on the FOSDEM site (well, the typo - the capitalization was right there already, but is not reroduced here for some reason). I was not sure whether changes on fosdem.org would be reflected automatically here.

Reply:
Well, I'll pull down the XML again at some point and re-run the script and send a new PR — and the updates should show up. The changes are broken into 3 regular expressions that uniquely fix the errors, so it would just not apply the fixes if they're not needed. It'll work out. :wink:

FOSDEM typically seems to update the XML export a few times a week... and it doesn't seem to immediately update.